src/CleanSolutionTemplate.Application/Common/Wrappers/IDateTimeOffsetWrapper.cs

```diff
@@ -2,5 +2,5 @@ namespace CleanSolutionTemplate.Application.Common.Wrappers;
 
 public interface IDateTimeOffsetWrapper
 {
-    DateTimeOffset UtcNow { get; }
+    DateTimeOffset UtcNow => DateTimeOffset.UtcNow;
```

Comment:
I think it's better if we leave the actual implementation of this one to the class instead of using the default implementation in the interface. Mainly because it's clearer that the infrastructure layer should be the one handling the actual implementation